fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraSession.java

```diff
@@ -36,9 +38,26 @@
     void expire();
 
     /**
+     * Save the session
+     *
+     * Note: use of the save method in the context of a batch operation may not persist
+     * changes to the repository until the commit method has been issued.
+     */
+    void save();
+
+    /**
      * Commit any batch operations
      */
-    void commit();
+    default void commit() {
```

Comment:
Yeah, I already made two; in one case, stop making batches and "sessions" the same supertype. No polymorphism. Completely independent types. In the other (preferably, to my mind), do not have two different types at all. After all, the distinction between them is based on "how many HTTP requests built this changeset? one, or more than one?" That's purely an HTTP-layer-concern. At the level of this code, we should just have one "session"y thing. How many HTTP requests built it up is of no concern down here.